src/main/python/deep_qa/layers/noisy_or.py

```diff
@@ -0,0 +1,88 @@
+from keras import backend as K
+from keras.layers import Layer
+from keras.initializations import identity
+from keras.constraints import Constraint, NonNeg
+from keras.regularizers import L1L2Regularizer
+from overrides import overrides
+
+class BetweenZeroAndOne(Constraint):
+    """Constrains the weights to be between zero and one
+    """
+    def __call__(self, p):
```

Comment:
I think I did this to be consistent with the way Keras does Constraints.  I can certainly change it if you want, or we can leave it for consistency?